src/grpc/helium_stream_sc_follow_impl.erl

```diff
@@ -28,71 +28,49 @@
         SCLastBlockTime :: non_neg_integer()
     }.
 
--record(handler_state, {
-    %% tracks which SC we are following
-    sc_follows = #{} :: #{binary() => follow()},
-    %% tracks which SCs we have send a closed msg for
-    sc_closes_sent = [] :: list(),
-    %% tracks which SCs we have send a closing msg for
-    sc_closings_sent = [] :: list(),
-    %% tracks which SCs we have send a closable msg for
-    sc_closables_sent = [] :: list(),
-    %% tracks which SCs we have send a dispute msg for
-    sc_disputes_sent = [] :: list()
-}).
+-type handler_state() :: #{
```

Comment:
Why did you switch from a record to a map? Is this a situation where the internal keys will need to ever be serialized repeatably externally? (We will need to sort the keys in that case.)

Reply:
switched to maps for convenience of accessing state in each of the different handlers.  The helium_gateway_service module needs to access the state in order to determine which modules to route info msgs to ( see https://github.com/helium/sibyl/blob/andymck/poc-grpc/src/grpc/helium_gateway_service.erl#L90 )

The use here is solely for internal process state